config/default/kustomization.yaml

```diff
@@ -1,12 +1,5 @@
 # Adds namespace to all resources.
-namespace: governance-policy-addon-controller-system
-
-# Value of this field is prepended to the
-# names of all resources, e.g. a deployment named
-# "wordpress" becomes "alices-wordpress".
-# Note that it should also match with the prefix (text before '-') of the namespace
-# field above.
-namePrefix: policy-addon-ctrl-
+namespace: open-cluster-management
```

Comment:
LGTM. I think the goal here was to confirm that it could be deployed to alternate namespaces, but I suppose it's been working for so long and it probably isn't a very common practice, it's simpler to set it to the default.